File: nodejs/googletravel.js
const sdk = require('api')('@smartproxy/v1.0#25e7913l1ow524w');

sdk.auth('SPusername', 'SPassword');
sdk.realTimeExample({target: 'google_travel_hotels', query: 'hilton', parse: false})
  .then(res => console.log(res))
  .catch(err => console.error(err));

File: python/googletravel.py
import requests

task_params = {
    'target': 'google_travel_hotels',
    'query': 'hilton',
    'parse': False,
    'geo': 'London,England,United Kingdom'
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scraper-api.smartproxy.com/v2/scrape',
    json = task_params,
    auth = (username, password)
)

print(response.text)
